ui/webui/test/machine_install.py

```diff
@@ -87,7 +96,7 @@ def start(self):
                 "-device virtio-net-pci,netdev=hostnet0,id=net0,addr=0x4' "
                 f"--initrd-inject {os.getcwd()}/test/ks.cfg "
                 "--extra-args 'inst.ks=file:/ks.cfg' "
-                "--disk size=15,format=qcow2 "
+                f"--disk path={disk_image},bus=virtio,cache=unsafe,format=qcow2 "
```

Comment:
Since it's not the virt-install creating the image anymore, you can remove format parameter for the --disk.

Reply:
I'll fix that.

Reply:
Done